services/s3/src/main/java/software/amazon/awssdk/services/s3/internal/crt/DefaultS3CrtAsyncClient.java

```diff
@@ -396,6 +419,10 @@ public void afterMarshalling(Context.AfterMarshalling context,
                             executionAttributes.getAttribute(SdkInternalExecutionAttribute.REQUEST_CHECKSUM_CALCULATION))
                        .put(RESPONSE_CHECKSUM_VALIDATION,
                             executionAttributes.getAttribute(SdkInternalExecutionAttribute.RESPONSE_CHECKSUM_VALIDATION))
+                       .put(S3InternalSdkHttpExecutionAttribute.RESPONSE_FILE_PATH,
+                            executionAttributes.getAttribute(RESPONSE_FILE_PATH))
+                       .put(S3InternalSdkHttpExecutionAttribute.RESPONSE_FILE_OPTION,
+                            executionAttributes.getAttribute(RESPONSE_FILE_OPTION))
```

Comment:
Duplicate?

Reply:
Can customers configure this attribute? What's the default behavior? Is it the same as what we have now?

Reply:
Created a backlog item to track that.